src/components/atoms/AnyWayButton.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react';
 import { Button, ButtonProps } from '@material-ui/core';
 
 interface IProps extends ButtonProps {
-  onClick: () => any;
+  onClick: ( arg0: any ) => any;
```

Comment:
Use: `onClick: Function`

Reply:
Tried that. throws error for incorrect extension of ButtonProps.